src/map/clif.c

```diff
@@ -9851,21 +9851,37 @@ static int clif_guess_PacketVer(int fd, int get_previous, int *error)
 		err = 0;\
 		if( error )\
 			*error = 0;\
+		if( diskey ) ShowWarning("Client use no encryption while server do, please fix me\n"); \
```

Comment:
No, well it's misleading. What I want to inform here is that 1 client is in mistmatch with the server.
To fix it you can either use a client without obfuscation
or recompile without PACKET_OBFUSCATION.
Now with the later the client that currently well setup will have an issue.
I originally tought about changing this at runtime before realising that I shouldn't. What I want to achieve here is to help detect the mistmatch not to tell him which way to go.